client/css/base/_tables.css

```diff
@@ -1,20 +1,33 @@
 table {
-    overflow: hidden;
+  overflow: hidden;
+  width: 100%;
+  overflow-x: auto;
+  display: block;
 }
 
-td, th {
-    border: 1px solid #DADFE0;
-    padding: 5px;
+td,
+th {
+  width: 400px;
+  border-bottom: 1px solid $light-grey;
+  padding: 5px;
 }
 
 th {
-    text-align: center;
+  text-align: left;
 }
 
 th.left {
-    text-align: left;
+  text-align: left;
+}
+
+tr:hover,
+td:hover {
+  background: rgba(255, 255, 255, .8);
```

Comment:
I can see you store colors in variables everywhere else, could you do that here as well?